install.sh

```
@@ -527,24 +569,8 @@ wait_for_local_ai_health() {
-            sleep $check_interval
-            continue
-        fi
-        
-        # Check 2: Are models loaded? (check logs for success message)
-        if docker logs local_ai_server 2>&1 | grep -q "Enhanced Local AI Server started on ws://"; then
             echo ""  # Clear the progress line
-            print_success "✅ local-ai-server is ready and listening on port 8765"
+            print_success "✅ local_ai_server is ready and listening on port 8765"
             print_info "Models loaded successfully (verified from logs)"
             return 0
         fi
```

Shellcheck (0.11.0)

[error] Expected a '}'. If you have one, try a ; or \n in front of it.

(SC1056)